### DISCUSSION TITLE
How to add posts,categories and tags to content section in manager?

### DISCUSSION THREAD

**Discussion by Gildar76** [PiranhaCMS/piranha.core#1991] [Q&A] [answered]
Hi,

I'm trying out Piranha for a simple blog website using Postgresql.

I installed the templates and got the empty,modules,razor and MVC templates (Note: no blog template)

Since the Piranha blog template no longer seems to exist. I tried created both an MVC web app and A razor web app.

Running the app with Postgresql works fine after adding a line for legacy behaviour for datetime in both cases.

Generating sample content works fine.

However, when I go into the manager.  There are no way to manage posts, categories and tags (along with a few other things such as language). Looking at the roles section I can see them there, and permissions looks fine.

My question is, how do I add these to the manager? Is there an example somewhere to do this?

Thanks
Fredrik

**Comment by tidyui** [comment, marked as answer]
All post related stuff is handled in the respective archive. Click on the Blog page in the manager and select the tab “Archive”.

Regards

**Reply by Gildar76** [reply]
Thank you very much. Not sure why I didn't click that.
